.github/workflows/production_pipeline.yml

```diff
@@ -48,7 +50,7 @@ jobs:
       - name: Get latest release with tag
         id: latestrelease
         run: |
-          echo "::set-output name=releasetag::$(curl -s https://api.github.com/repos/${REPO_URL}/releases/latest | jq '.tag_name' | sed 's/\"//g')"
+          echo "releasetag=$(curl -s https://api.github.com/repos/${REPO_URL}/releases/latest | jq '.tag_name' | sed 's/\"//g')" >> $GITHUB_OUTPUT
```

Comment:
`set-output` has been deprecated, so I changed it to the preferred method (using `GITHUB_OUTPUT`) here and in some other places. There are other places in the pipelines where I haven't updated it - I thought better to raise this PR first, then if you agree with my thinking I can raise a ticket to clean up the rest of the pipelines later.

Reply:
I have raised 2 tickets for this:
- https://dluhcdigital.atlassian.net/browse/CLDC-2888
- https://dluhcdigital.atlassian.net/browse/CLDC-2889